apps/site/src/pages/api/remove-user-avatar.api.ts

```diff
@@ -0,0 +1,33 @@
+import { createAuthenticatedHandler } from "../../lib/api/handler/authenticated-handler";
+import { formatErrors } from "../../util/api";
+
+export type RemoveUserAvatarResponse = "SUCCESS";
+
+export default createAuthenticatedHandler<
+  unknown,
+  RemoveUserAvatarResponse
+>().get(async (req, res) => {
```

Comment:
I think this should be `delete` method, and it can be at `me/remove-avatar` rather than here (we have other stuff specific to the user at `me/`)